docker-compose.yml

```diff
@@ -102,7 +103,7 @@ services:
     mysql-integration-test:
         <<: *mysql-common
         ports:
-            - 3307:3306
+            - ${MYSQL_PORT}:3306
```

Comment:
I have 3307 set up in `phpunit.xml` because I have another MySQL server running on my machine, and don't want to use it for integration testing (different version, and rather use the Docker container). By default, it should stay 3306, and only use 3307 when the integration tests kick in.

Reply:
This should be fixed now as the test environment stuff and the docker stuff are not being used in the same process anymore